app/script/conversation/ConversationRepository.js

```diff
@@ -3523,7 +3523,7 @@ z.conversation.ConversationRepository = class ConversationRepository {
    */
   _delete_message(conversation_et, message_et) {
```

Comment:
Seems like we have to build a proper EventRepository in the next step to move the event service related functions out of the ConversationRepository. Perfect start to slim that one down as well.

Reply:
I cannot agree more with that statement. That said, we already have an `EventRepository` which is... not really small :D

Reply:
That one should get a different name. It's more of a `NotificationRepository` - which we also have. So maybe a `InboundBackendTrafficNotificationHandlingRepository`